data/org.sugarlabs.gschema.xml

```diff
@@ -87,6 +87,11 @@
             <summary>Group Label</summary>
             <description>Label associated with age, e.g., '2nd Grade'</description>
         </key>
+        <key name="resume-activity" type="b">
+            <default>false</default>
```

Comment:
Setting it to false means that "start new" is the default mode.  This is a change from normal sugar.

I'm fine with adding this setting, please set the default to "true" though.

If you want to have the default as "false" that is a design change.  That would need to be discussed.